source/renderer/app/components/wallet/transactions/WalletTransactionsList.js

```diff
@@ -56,6 +56,7 @@ type Props = {
   onOpenExternalLink?: Function,
   showMoreTransactionsButton?: boolean,
   transactions: Array<WalletTransaction>,
+  locale: string,
```

Comment:
@DeeJayElly I find this prop name confusing. Let's rather use `dateLocale` instead 🙏

Reply:
Please do that on all other places too 🙏

Reply:
@nikolaglumac done